storage/key

```diff
@@ -1 +1 @@
-$2y$10$WabkIv3ZDE5eWrZqn87YRegG/LH2GnFoIy92RFSQHMQF4JQegqGIS
\ No newline at end of file
+$2y$10$fFeWc6kN/wkwnCgxQSWFOu1H2f36U96hfMedatMueLVe4D1BBFU2G
```

Comment:
Agree - I was quite shocked it was not in the .gitignore too (I didn't check the git status message carefully)